client/my-sites/plugins/marketplace/marketplace-domain-upsell/index.tsx

```diff
@@ -0,0 +1,171 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import DomainPicker from '@automattic/domain-picker';
+import { useShoppingCart } from '@automattic/shopping-cart';
+import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
+import { domainRegistration } from 'calypso/lib/cart-values/cart-items';
+import { DomainSuggestions } from '@automattic/data-stores';
+import {
+	MARKETPLACE_FLOW_ID,
+	ANALYTICS_UI_LOCATON_MARKETPLACE_DOMAIN_SELECTION,
+} from 'calypso/my-sites/plugins/marketplace/constants';
+import { Button } from '@wordpress/components';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+import './styles.scss';
+import { getProductsList, isProductsListFetching } from 'calypso/state/products-list/selectors';
```

Comment:
Nit: This seems to belong above, with the rest of the internal imports.

Also: I've seen styles being imported with this doc block above them, in case this is something you'd like to use:

```
/**
 * Style dependencies
 */
```

Reply:
Fixed by 0aa07864d5e9a9480c7959ac92405d761970af1e

Reply:
Fixed by 0aa0786